patches/react-native+0.64.0.patch

```diff
@@ -23,3 +23,27 @@ index 52cbda4..3e3b71e 100644
    hidesWhenStopped: true,
    size: 'small',
  };
+diff --git a/node_modules/react-native/Libraries/Image/AssetSourceResolver.js b/node_modules/react-native/Libraries/Image/AssetSourceResolver.js
+index 5ebd97a..3f511b9 100644
+--- a/node_modules/react-native/Libraries/Image/AssetSourceResolver.js
++++ b/node_modules/react-native/Libraries/Image/AssetSourceResolver.js
+@@ -92,9 +92,18 @@ class AssetSourceResolver {
+    */
+   assetServerURL(): ResolvedAssetSource {
```

Comment:
I believe this patch will only repair the Metro server used for local development. From my testing, the static image assets are missing when the bundle is integrated into WPAndroid and WPiOS. So, we will need to address the bundle assets in a separate PR in the future. 

That said, I believe I have found an alternative Metro server fix that is less invasive. It relies upon changes to the local `metro.config.js` file instead of patching third-party code. I plan to open a PR against this branch for your consideration.

Reply:
https://github.com/WordPress/gutenberg/pull/33829 is the follow up PR for consideration.

Reply:
That's great news! I'll take a look at your PR and hold of merging this one until your PR is merged into this branch. 👍